tengine/jae/adding_template_partials.md

```diff
@@ -0,0 +1,141 @@
+# Adding Template Partials
+
+## Table of Contents
+
+* [Before You Get Started](#before-you-get-started)
+* [Folder Structure End State](#folder-structure-end-state)
+* [Steps](#steps)
+* [Verification](#verification)
+
+## Before You Get Started
+
+In this tutorial, you will create an HTML template with template partials using `<% include_partial("/skills.html") %>`.
+
+## Folder Structure End State
+
+Upon completing the Steps section below, your project's folder structure should look similar to the following:
+
+```
+▾ /path/to/your/project/
+  ▾ /views
+    skills.html
+    user.html
+  app.ts
+  user_resource.ts
+```
+
+## Steps
+
+1. Create your template file that will include the template partial. _Note: The `skills.html` file must be relative to the `views_path` config.
```

Comment:
The note section is supposed to be in italics I’m guessing, but it’s missing a closing ‘_’

Reply:
eb17b7a